### PR TITLE
Syntax fixes.

### PR DESCRIPTION
**Pull request by SaracenOne**
Fixes some syntax errors with the exporter.

### CODE REVIEW

File: io_scene_dae/export_dae.py [hunk 1]
```diff
@@ -188,19 +188,15 @@ def export_image(self, image):
                 # folder
                 img_tmp_path = image.filepath
                 if img_tmp_path.lower().endswith(bpy.path.extensions_image):
-                    image.filepath =
-                        os.path.join(basedir, os.path.basename(img_tmp_path))
+                    image.filepath = os.path.join(basedir, os.path.basename(img_tmp_path))
```
Comment (akien-mga) [inline io_scene_dae/export_dae.py]: This reverts some changing done to ensure PEP-8 compliance. To be honest I prefer the one line form a thousand times, though I understand the need and advantages of following an established style guide.

**Comment by SaracenOne** [comment]
The only problem is that for whatever reason, my version of Blender wouldn't recognise that type of formatting.

**Comment by akien-mga** [comment]
CC @paper-pauper @set-killer

**Comment by set-killer** [comment]
True, it is not valid syntax. This fixes are implemented in the other merge request, which i've submitted an hour ago.

**Comment by SaracenOne** [comment]
I just checked out @set-killer's PR, and it contains all the same fixes here, so we might as well close this one.

**Pull request closed without merging**